kraken/csv.go

```diff
@@ -102,6 +101,11 @@ func (kr *Kraken) ParseCSV(reader io.Reader) (err error) {
 					t.Items = make(map[string]wallet.Currencies)
 					t.Items["From"] = append(t.Items["From"], wallet.Currency{Code: tx.Asset, Amount: tx.Amount.Neg()})
 					kr.TXsByCategory["Withdrawals"] = append(kr.TXsByCategory["Withdrawals"], t)
+				} else if tx.Type == "staking" {
+					t := wallet.TX{Timestamp: tx.Time}
+					t.Items = make(map[string]wallet.Currencies)
+					t.Items["Interests"] = append(t.Items["Interests"], wallet.Currency{Code: tx.Asset, Amount: tx.Amount})
```

Comment:
Tu n'as pas mis les Items

Reply:
Effectivement, my bad!
J'ai aussi rajouté les fees et je pense que c'est bon, encore deux trois tests et je passe ça en review

Reply:
pour tes tests avec CSV réel, tu utilises le -check pour vérifier que tu ne créé par de TX malformé (montant négatif et Transfers à somme non nulle) ? (je vais ajouter des check qui sont propre a la philosophie de l'outil)
Je n'ai pas de compte Kraken et donc je ne peux pas vérifier cela.

Reply:
Yes, en testant sans mes autres comptes (binances, etherscan etc..) et avec, "List of Non-Zero balance Transfers" et "List of Negative Amounts TXs" sont bien vide.
Seulement "List of Unjustified Withdrawals (after 2019 Jan 1st)" se rempli sans les autres comptes.
Si besoin, je devrais pouvoir t'envoyer un csv généré par Kraken et "anonymisé"

Reply:
ok cool
merci pour ton taf en tout cas, tu as l'air bien compétent, tu veux pas venir sur le groupe telegram pour discuter ?